grammar.js

```diff
@@ -424,7 +424,11 @@ module.exports = grammar({
     use: ($) =>
       seq(
         "use",
-        optional(series_of($._pattern, ",")),
+        optional(series_of(
+          seq(
+            $._pattern,
+            optional($._type_annotation)
+          ), ",")),
```

Comment:
Looks like this needs a run of the formatter. If you use `npm format` it should run prettier and fix the CI